### ISSUE TITLE
`<tspan>` doesn't include a text node (`permitted-contents`)

### ISSUE THREAD

**Issue by YusukeHirao** [markuplint/markuplint#1359]
- Markuplint version: `4.0.0`
- Parser lang: HTML or JSX and more...
- Node.js version: `20.x`
- OS: macOS or Windows

## Code Example

```html
<svg>
 <text><tspan>Text</tspan></text>
</svg>
```

## The current behavior

> The text node is not allowed in the "tspan" element in this context (permitted-contents)

## The expected behavior

No error
